src/pool-bin/BinPoolManager.sol

```diff
@@ -291,6 +300,14 @@ contract BinPoolManager is IBinPoolManager, ProtocolFees, Extsload {
         emit SetMaxBinStep(maxBinStep);
     }
 
+    /// @inheritdoc IBinPoolManager
+    function setMinBinSharesForDonate(uint256 minBinShare) external override onlyOwner {
+        if (minBinShare < 1e18) revert MinShareTooSmall();
```

Comment:
open to removing this check even, its just a mini sanity check in case we call with `0` or forget its supposed to be a large number